pkgs/development/compilers/vlang/default.nix

```diff
@@ -30,8 +30,8 @@ let
   markdown = fetchFromGitHub {
     owner = "vlang";
     repo = "markdown";
-    rev = "61c47ea0a6c0c79e973a119dcbab3b8fdd0973ca";
-    hash = "sha256-XBD30Pc9CGXzU1Gy6U0pDpTozYVwfgAvZRjIsnXp8ZM=";
+    rev = "0c280130cb7ec410b7d21810d1247956c15b72fc";
+    hash = "sha256-Fmhkrg9DBiWxInostNp+WfA3V5GgEIs5+KIYrqZosqY=";
```

Comment:
Same as last comment.
Not sure if it applies, but would be nice if we could pin to the release version as well.

Reply:
> Same as last comment. Not sure if it applies, but would be nice if we could pin to the release version as well.

This one (markdown) is more difficult to pin as there isn't a commit on the same date. Just one 2 weeks before, and one a week after. But the commit the week after (which is currently the latest) references the new version of V - "Compile cleanly with the latest V", so I've pinned that one (i.e. the latest).